.github/workflows/test_merge_requests.yml

```diff
@@ -150,9 +151,18 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
 
-      - name: 'Run tests'
-        env:
-          CI: true
-        run: |
+      - name: Run tests
+        run: >
           NAME=mailserver-testing:ci
           make generate-accounts tests
+        env:
+          CI: true
+
+  job-build-arm:
+    name: Build ARM Image
+    needs: job-build-image
+    uses: ./github/workflows/generic_build.yml
+    with:
+      platforms: linux/arm/v7,linux/arm64
+      publish: false
+      cache-layers-to: false
```

Comment:
This may be problematic for cache if we drop the conditional handling.

Since it is a 13 minute build without cache, is it useful to cache?

Or do we not care about this as I think we're primarily using the cache for AMD64 to speed along PRs (_especially iterating from test suite feedback_)?

If we care about caching it, then with old and new approaches to deriving a cache key, this would conflict for the same cache key but different data.

We can include `-amd64-` into the the existing cache key before the digest suffix value. Then if this were to be cached using `-arm-` instead would resolve that.

Without `type=gha` for buildx cache exporter though, it seems really messy to support multiple build caches.

Reply:
> building the ARM image is now done after the initial build to run in
parallel with BATS.

Actually, since it defers until this point, we should always have the `AMD64` image built and cached, so this should be a non-issue, it can use the same cache since it shouldn't ever need to upload for a new cache key? (_when `key` is exact match, it'll skip upload_)

Reply:
👍🏼